src/models/jwtPermissions/index.js

```diff
@@ -136,3 +155,48 @@ exports.init = async function ({ publicKeyUrl = _publicKeyUrl, loadPublicKeyFrom
         }
     }, 60 * 60 * 1000);
 };
+
+/**
+ *
+ * @param {string}tokenMerchantId
+ * @param {string}uriMerchantId
+ * @returns {void}
+ */
+const validateMerchant = (tokenMerchantId, uriMerchantId) => {
+    if (tokenMerchantId !== uriMerchantId && tokenMerchantId !== '*') {
```

Comment:
we should also validate placeId if it is required by the app
(the route has placeId parameter or a placeId is provided by argument authorizeUser function)

in that case we should validate the placeIds in the token includes the id or '*'